src/followers/render.php

```diff
@@ -0,0 +1,153 @@
+	'followers' => $followers,
+	'total'     => $follower_data['total'],
+	'pages'     => ceil( $follower_data['total'] / $_per_page ),
+	'isLoading' => false,
+);
+
+// Get block wrapper attributes with the data-wp-interactive attribute.
+$wrapper_attributes = get_block_wrapper_attributes(
+	array(
+		'id'                  => wp_unique_id( 'activitypub-followers-block-' ),
+		'data-wp-interactive' => 'activitypub/followers',
+		'data-wp-context'     => wp_json_encode( $context, JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
+	)
+);
+?>
+
+<div <?php echo $wrapper_attributes; // phpcs:ignore WordPress.Security.EscapeOutput ?>>
+	<?php echo $content; // phpcs:ignore WordPress.Security.EscapeOutput ?>
+
+	<div class="followers-container">
+		<ul class="followers-list">
+			<template data-wp-each="context.followers">
+				<li class="follower-item">
+					<a data-wp-bind--href="context.item.url"
+						class="follower-link"
+						target="_blank"
+						rel="external noreferrer noopener"
+						data-wp-bind--title="context.item.handle">
+
+						<img
+							data-wp-bind--src="context.item.icon.url"
+							data-wp-on--error="callbacks.setDefaultAvatar"
+							src=""
+							alt=""
+							class="follower-avatar"
+							width="48"
+							height="48"
+						>
+
+						<div class="follower-info">
+							<span class="follower-name" data-wp-text="context.item.name"></span>
+							<span class="follower-username" data-wp-text="context.item.handle"></span>
+						</div>
+
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24" class="external-link-icon" aria-hidden="true" focusable="false" fill="currentColor">
+							<path d="M18.2 17c0 .7-.6 1.2-1.2 1.2H7c-.7 0-1.2-.6-1.2-1.2V7c0-.7.6-1.2 1.2-1.2h3.2V4.2H7C5.5 4.2 4.2 5.5 4.2 7v10c0 1.5 1.2 2.8 2.8 2.8h10c1.5 0 2.8-1.2 2.8-2.8v-3.6h-1.5V17zM14.9 3v1.5h3.7l-6.4 6.4 1.1 1.1 6.4-6.4v3.7h1.5V3h-6.3z"></path>
+						</svg>
+					</a>
+				</li>
+			</template>
+		</ul>
+
+		<?php if ( $follower_data['total'] > $_per_page ) : ?>
+		<nav class="followers-pagination" role="navigation">
+			<h1 class="screen-reader-text"><?php esc_html_e( 'Follower navigation', 'activitypub' ); ?></h1>
+			<button
+				class="pagination-previous wp-block-button__link wp-element-button"
+				data-wp-on--click="actions.previousPage"
+				data-wp-bind--disabled="state.disablePreviousButton"
+				aria-label="<?php esc_attr_e( 'Previous page', 'activitypub' ); ?>"
+				disabled
+			>
+				<?php esc_html_e( 'Previous', 'activitypub' ); ?>
+			</button>
+
+			<div class="pagination-info" data-wp-text="state.paginationText"></div>
+
+			<button
+				class="pagination-next wp-block-button__link wp-element-button"
+				data-wp-on--click="actions.nextPage"
+				data-wp-bind--disabled="state.disableNextButton"
+				aria-label="<?php esc_attr_e( 'Next page', 'activitypub' ); ?>"
+			>
+				<?php esc_html_e( 'Next', 'activitypub' ); ?>
+			</button>
+		</nav>
```

Comment:
this part is now defined in several places and makes it hard to update.

Reply:
anything that might help that these will not run "out of sync"? (that we only update one of the different related sites?)